tests/dummy_app/Gemfile

```diff
@@ -17,6 +14,7 @@ gem 'sqlite3'
 gem 'puma', '~> 3.7'
 # Use SCSS for stylesheets
 gem 'sass-rails', '~> 5.0'
+gem 'bootstrap-sass'
```

Comment:
We really don't need Bootstrap here to provide RSpec tests. I think, you've done it just because you're used to add it to your projects. I don't like that we have it here. Please remove it.

Нам в действительности не нужен бутстрап для наших rspec тестов. Я думаю, ты сделал это потому что просто привык добавлять его в свои проекты. Пожалуйста удали его.